fastchat/client/openai_api_client.py

```diff
@@ -62,9 +62,13 @@ async def request_completion_stream(
                 async for chunk in response.aiter_text():
                     if not chunk:
                         continue
-                    for line in chunk.split("\n"):
+                    for line in chunk.split("\n\n"):
```

Comment:
elaborate why `\n\n` here instead of `\n`

Reply:
I follow the [Event Stream Format](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#event_stream_format) - Messages in the event stream are separated by a pair of newline characters.